dev-tools.asciidoc

```diff
@@ -358,10 +358,53 @@ $ npm -g install embark
 ----
 
 === OpenZeppelin
-OpenZeppelin is an open framework of reusable and secure smart contracts in the Solidity language.
-OpenZeppelin integrates well with Truffle
 
-Github link; https://github.com/OpenZeppelin
+https://openzeppelin.org/[OpenZeppelin] is an open framework of reusable and secure smart contracts in the Solidity language.
+
+It is community-driven, lead by the https://zeppelin.solutions/[Zeppelin] developers, with hundreds of contributors from many Ethereum projects. The main focus of the framework is security, achieved by applying industry standard contract security patterns and best practices, taking advantage of all the experience the Zeppelin devs have gained from https://blog.zeppelin.solutions/tagged/security[auditing] a huge amount of contracts, and through the constant testing and auditing from the community that uses the framework as a base for their real world applications.
```

Comment:
hundreds of contributors from many Ethereum projects => with over a hundred external contributors.